src/Pact/Types/Info.hs

```diff
@@ -82,6 +83,10 @@ instance Ord Info where
   Info Nothing <= _ = True
   _ <= Info Nothing = False
 
+instance Mergeable Info where
```

Comment:
Since you have an Orphans file already I might suggest you move this there, as we might seek to take the SBV dependency out of blockchain-targeted builds at some point.

Reply:
fixed